Sources/ProjectDescription/Dependencies/SwiftPackageManagerDependencies.swift

```diff
@@ -15,27 +30,27 @@ public struct SwiftPackageManagerDependencies: Codable, Equatable {
     public let targetSettings: [String: SettingsDictionary]
 
     /// Custom project configurations to be used for projects generated from SwiftPackageManager.
-    public let configurations: [String: ProjectDescription.Project.Configuration]
+    public let generationOptions: [String: ProjectDescription.Project.Options]
```

Comment:
This should be `projectOptions` to avoid confusion between config options, workspace options, and target options :D